tests/countries/test_finland.py

```diff
@@ -167,12 +167,17 @@ def test_all_saints_day(self):
             "1957-11-01",
         )
 
+    def test_independence_day(self):
+        name = "Itsenäisyyspäivä"
+        self.assertHolidayName(name, (f"{year}-12-06" for year in range(1917, 2050)))
+        self.assertNoHolidayName(name, (f"{year}-12-06" for year in range(1853, 1917)))
```

Comment:
```suggestion
        self.assertNoHolidayName(name, range(1853, 1917))
```

Reply:
Doing this somehow reduces test coverage for Finland, I'm looking at the codes at the moment why this happens.

Reply:
I expanded my suggestion.